packages/web/pages/apps/index.tsx

```diff
@@ -0,0 +1,163 @@
+import Fuse from "fuse.js";
+import React from "react";
+import { useTranslation } from "react-multi-lang";
+
+import { Button } from "~/components/buttons";
+import { HeroCard } from "~/components/cards";
+import AppDisplayCard from "~/components/cards/app-display-card";
+import { SearchBox } from "~/components/input";
+import { EventName } from "~/config";
+import { useAmplitudeAnalytics } from "~/hooks";
+
+type appDataType = {
+  title: string;
+  subtitle: string;
+  thumbnail_image_URL: string;
+  external_URL: string;
+  twitter_URL?: string;
+  github_URL?: string;
+  medium_URL?: string;
+  featured?: boolean;
+};
+
+type App = {
+  title: string;
+  subtitle: string;
+  external_URL: string;
+  thumbnail_image_URL: string;
+  hero_image_URL: string;
+  twitter_URL?: string;
+  medium_URL?: string;
+  github_URL?: string;
+  featured?: boolean;
+};
+
+type AppStoreProps = {
+  apps: {
+    applications: App[];
+  };
+};
+
+export const AppStore: React.FC<AppStoreProps> = ({ apps }) => {
+  const [searchValue, setSearchValue] = React.useState<string>("");
+  const [fuzzySearchResults, setFuzzySearchResults] = React.useState<
+    appDataType[]
+  >([]);
+
+  const { applications } = apps;
+
+  const t = useTranslation();
+  const { logEvent } = useAmplitudeAnalytics();
+
+  let featuredApp = applications?.find((app) => app.featured === true);
+  const nonFeaturedApps = applications?.filter((app) => !app.featured);
+
+  featuredApp = featuredApp ? featuredApp : nonFeaturedApps[0];
+
+  const options = {
+    keys: ["title"],
+  };
+  const fuse = new Fuse(applications, options);
```

Comment:
Heads up we have a useFilteredData hook in use-filtered-data.ts that will manage all of the state and will maintain an instance of Fuse as long as possible so we can get its caching benefits as long as the data doesn't change. Overkill for the amount of data, but still worth mentioning.

Periodically, I recommend looking in the hooks folder for utils that may be useful.
Also, we import `react-use` that also has some helpful links.